build.mjs

```diff
@@ -12,6 +12,7 @@ async function build(path) {
     packages: 'external',
     bundle: true,
     sourcemap: true,
+    format: 'cjs',
```

Comment:
I didn't realise `'cjs'` was not the default option, instead it's `'iife'`:
https://esbuild.github.io/api/#format-iife

Reply:
Example output diff

<img width="1775" alt="image" src="https://github.com/radix-ui/primitives/assets/8441036/7148691d-295e-4b39-8374-f10778300002">